Sources/DistributedCluster/Cluster/SWIM/SWIMActor.swift

```diff
@@ -355,6 +355,13 @@ internal distributed actor SWIMActor: SWIMPeer, SWIMAddressablePeer, CustomStrin
             )
         }
 
+        /// If SWIM claims we are dead, ignore this; we should be informed about this in high-level gossip soon enough.
+        if change.status == .dead,
+           change.member.node.asClusterNode == self.id.node
+        {
+            return
+        }
```

Comment:
The fix™

Though this deserves a deeper dig into SWIM as well.

Also, SwiftFormat's formatting of multi-line if is annoying omg!